plugins/utils/comment-box.nix

```
@@ -0,0 +1,160 @@
+{
+  lib,
+  config,
+  pkgs,
+  ...
+}:
+let
+  inherit (lib)
+    mkAttrsOf
+    mkBool
+    mkEnum
+    mkInt
+    mkStr
+    types
+    ;
+in
+lib.nixvim.neovim-plugin.mkNeovimPlugin config {
+  name = "comment-box";
+  originalName = "comment-box.nvim";
+  luaName = "comment-box";
+  defaultPackage = pkgs.vimPlugins.comment-box-nvim;
+  description = ''
+    Clarify and beautify your comments and plain text files using boxes and lines.
+  '';
+
+  maintainers = [ lib.maintainers.elythh ];
+
+  settingsOptions = {
+    comment_style =
+      mkEnum
+        [
+          "line"
+          "block"
+          "auto"
+        ]
+        "line"
+        ''
+          Select the type of comments. Default: 'line'.
+        '';
+
+    doc_width = mkInt types.number 80 ''
```

We should make it more clear where these are coming from to fit in line with our other usage across codebase.This will apply to all options, just showing suggestion on this one to avoid repetition. 

```suggestion
    doc_width = defaultNullOpts.mkInt types.number 80 ''
```

---

Done https://github.com/nix-community/nixvim/compare/df79c0b795e86a4af80c342037bc4fd31dda95e0..a1761087fa9bbaef4cf95db6102f2c88ba48893b